src/models/entry.js

```diff
@@ -0,0 +1,14 @@
+export default class Entry{
+    constructor(hours, userId,UTCtime,jobType){
+
+        this.hours = hours;
+        this.userId = userId;
+        this.UTCtime = UTCtime;
```

Comment:
Is the UTCtime the start or end? Also did we want to keep track of both or calculate it based on hours as well

Reply:
So this is a straight pull from the erd. Can and will be updated. Talk tommorow at standup